src/api/appType/whatsapp_cloud/index.js

```diff
@@ -0,0 +1,14 @@
+import qs from 'query-string';
+import request from '@/api/request.js';
+
+const resource = '/api/v1/apptypes/wpp-cloud/apps';
+export default {
+  getWabaId(params) {
+    const queryString = qs.stringify(params);
+    return request.$http.get(`${resource}/waba_id/?${queryString}`);
+  },
```

Comment:
Did you try to define this method like this? 
```js
getWabaId: (params) => request.$http.get(`${resource}/waba_id/?${qs.stringify(params)}`);
```
and before it you can add:
```js
const stringfy = qs.stringify
```
and use the method like that:
```js
getWabaId: (params) => request.$http.get(`${resource}/waba_id/?${stringfy(params)}`);
```